spec/outputs/email_spec.rb

```diff
@@ -101,8 +101,61 @@
 
         expect(message_observer.messages.size).to eq(1)
         expect(message_observer.messages[0].subject).to eq("Hello World")
-        expect(message_observer.messages[0].body.decoded).to eq(craft_multi_part_email('', '<h1>hello</h1>', message_observer.messages[0].content_type))
```

Comment:
I had to change this test because it was wrong. It send in the `text/plain` content-type via `message_observer.messages[0].content_type` since before this patch we sent an empty `text/plain` mail even though we only wanted to send an `text/html` mail.